sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/DescribeTableSuite.scala

```diff
@@ -548,6 +547,9 @@ trait DescribeTableSuiteBase extends command.DescribeTableSuiteBase
                |""".stripMargin
           spark.sql(tableCreationStr)
           val viewType = if (isTemp) "TEMP VIEW" else "VIEW"
+          spark.sql("SET spark.sql.ansi.enabled = false") // non-default
+          spark.sql("SET spark.sql.parquet.enableVectorizedReader = true") // default
+          spark.sql("SET spark.sql.sources.fileCompressionFactor = 2.0") // non-default
```

Comment:
nit: in tests we should use `WithSQLConf` to set configs, so that it can restore the configs at the end. Otherwise these configs will affect other tests that use the same session.